org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/LanguageServerImpl.xtend

```diff
@@ -677,19 +686,21 @@ import static org.eclipse.xtext.diagnostics.Severity.*
 							val serviceProvider = resource.URI.resourceServiceProvider;
 							val coloringService = serviceProvider?.get(IColoringService);
 							if (coloringService !== null) {
```

Comment:
I think this needs to be removed since IColoringService is deprecated and then we don't need the binding in the TestLanguageModule anymore.

Reply:
> I think this needs to be removed

"this" what? Did you mean the`null` check or the entire logic?

Reply:
"this" means the entire logic but as mentioned above -> Ok. If someone has used it then we still need to call it in LS. fair enough.